app.go

```diff
@@ -37,6 +38,21 @@ type app struct {
 	c *client.Client
 }
 
+func (w *app) Activate(raiseAllWindows bool, ignoreOtherApps bool) error {
+
+	t := true
```

Comment:
Might be worth making a `func b(b bool) *bool` next to the already existing `func str`